starter_code/css/style.css

```diff
@@ -64,59 +62,51 @@ a {
 
 /* Center the container */
 .container {
-  width: 960px;
+  max-width: 960px;
   margin: 0 auto;
 }
 .logo {
-  font-size: 48px;
   font-weight: 800;
 }
 
 header {
   background: #033048;
-  padding: 25px 0;
   overflow: hidden;
 }
+
 header .logo {
   color: #fff;
   float: left;
 }
 
 header nav {
   float: right;
```

Comment:
Yes, I was referring to floats. And ur correct, they would be applied to your desktop automatically. The point is, you _don't_ want them applied to your mobile styles. Right now you effectively have this:

baseStyles.css (universal to all screens)
```
header nav {
  float: left;
}
```

mobile.css (screens from 320px wide and **UP**)
```css
header nav {
  float: none;
}
```

desktop.css (screens 768px and **UP**)
```css
header nav {
  float: left;
}
```

The problem is, ur saying all screens get the `float: left` behavior but in fact you only want it applied to your larger screens.

So in the above scenario, ur effectively saying ALL screens get the `float: left`, then ur saying "okay, mobile I wanna strip that behavior away", and then on desktop ur saying "oh wait, now I need to add that behavior back because the mobile styles are still be applied to desktop, and I want to enforce floating left".

So a better way to do it would be like this:

```
header nav {
  /* nothing here regarding floats */
}
```

mobile.css (screens from 320px wide and **UP**)
```css
header nav {
  /* still nothing here regarding floats */
}
```

desktop.css (screens 768px and **UP**)
```css
header nav {
  float: left; /* now that we're on a big boy screen, do the float stuff */
}
```